test/service/test_response.py

```diff
@@ -1397,5 +1397,38 @@ def test_summary_filter_none(self):
         self.assertEqual(summary_object['projectCount'], 2)
         self.assertEqual(summary_object['totalCellCount'], 6210.0)
 
+class TestIntegrationRepsonse(WebServiceTestCase):
+    maxDiff = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls._setup_indices()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._teardown_indices()
+        super().tearDownClass()
+
+    def test_integrations(self):
+        def request_integration(integration_type, entity_type):
+            url = self.base_url + f'/integrations?integration_type={integration_type}&entity_type={entity_type}'
+            response = requests.get(url)
+            response.raise_for_status()
+            return response.json()
+
+        test_data = [
+            ('get_manifest', 'file', 1),
+            ('get', 'project', 9),
+        ]
+        for integration_type, entity_type, integrations in test_data:
+            with self.subTest(integration_type=integration_type, entity_type=entity_type, integrations=integrations):
```

Comment:
```suggestion
            with self.subTest(integration_type=integration_type, entity_type=entity_type, num_integrations=num_integrations):
```

and possibly line wrap after each argument